controllers/user.go

```diff
@@ -0,0 +1,25 @@
+package controllers
+
+// Createuser will receive the users information as a request
+//  and then store the Users information in the user model in the MySQL Database Server
+func createUser() {
```

Comment:
All exported functions in Go need to be capitalized. We will be calling these functions from `package transport` and they exist in `package controllers`. Therefore, they need to be capitalized.